### PR TITLE
Move error messages into resource files

### PR DESCRIPTION
**Pull request by mikary**
Moving string literal error messages discovered in #2752 into appropriate resource files

### CODE REVIEW

**Comment by lajones** [comment]
:shipit: Some of the error messages in ExpressionPrinter don't seem very useful (e.g. about what you should do next) but if they're just meant for debug or are made clear by the context in which they're issued then no problem.

File: src/EntityFramework.Commands/Design/Internal/CommandLogger.cs [hunk 1]
```diff
@@ -85,7 +86,7 @@ public virtual IDisposable BeginScope([NotNull] object state)
                     WriteDebug(message.ToString());
                     break;
                 default:
-                    Debug.Fail("Unexpected event type: " + logLevel);
+                    Debug.Fail( CommandsStrings.UnexpectedLogEvent(logLevel));
```
Comment (bricelam) [inline src/EntityFramework.Commands/Design/Internal/CommandLogger.cs]: Debug strings shouldn't be resources. They'll just be a waste of space on Release builds.

Comment (Eilon) [inline src/EntityFramework.Commands/Design/Internal/CommandLogger.cs]: :+1:

**Comment by mikary** [comment]
I believe the error messages are generally intended to be useful for debugging. We should also have a chance to revisit them in API review now that they are included in the resources.

**Pull request merged**